src/js/BookReader/Mode1Up.js

```diff
@@ -226,38 +304,39 @@ export class Mode1Up {
       // current index in drawLeafsOnePage after we create the new view container
 
       // Make sure this will count as current page after resize
-      const fudgeFactor = (this.book.getPageHeight(this.br.currentIndex()) / this.br.reduce) * 0.6;
+      const fudgeFactor = 0.6 * this.physicalInchesToDisplayPixels(this.book.getPage(this.br.currentIndex()).heightInches);
       const oldLeafTop = this.getPageTop(this.br.currentIndex()) + fudgeFactor;
-      const oldViewDimensions = this.calculateViewDimensions(this.br.reduce, this.br.padding);
+      const oldViewDimensions = this.calculateViewDimensions();
       scrollRatio = oldLeafTop / oldViewDimensions.height;
     }
 
     // Recalculate 1up reduction factors
     this.calculateReductionFactors();
     // Update current reduce (if in autofit)
     if (this.br.onePage.autofit) {
-      const reductionFactor = this.br.nextReduce(this.br.reduce, this.br.onePage.autofit, this.br.onePage.reductionFactors);
-      this.br.reduce = reductionFactor.reduce;
+      const reductionFactor = this.br.nextReduce(this.realWorldReduce, this.br.onePage.autofit, this.br.onePage.reductionFactors);
+      this.realWorldReduce = reductionFactor.reduce;
     }
 
-    const viewDimensions = this.calculateViewDimensions(this.br.reduce, this.br.padding);
-
-    this.br.refs.$brPageViewEl.height(viewDimensions.height);
-    this.br.refs.$brPageViewEl.width(viewDimensions.width);
+    const viewDimensions = this.calculateViewDimensions();
+    this.$documentContainer.height(viewDimensions.height);
+    this.$documentContainer.width(viewDimensions.width);
 
+    // Remove all pages
+    this.$documentContainer.empty();
+    this.br.displayedIndices = [];
```

Comment:
this signals to me a fresh draw on resize, how expensive is this when we view BR in context of book navigator & the menu slides open/close? (does it matter?)

Reply:
i am here for it, i think the image caching work has allowed for a "fresher" dom redraw at the moment since we don't have to account for an expensive request